.ci.yaml

```diff
@@ -4181,3 +4181,17 @@ targets:
       tags: >
         ["devicelab", "android", "windows"]
       task_name: windows_chrome_dev_mode
+
+  - name: Windows windows_startup_test
```

Comment:
@yaakovschectman previously named a test to contain `win_desktop`. You two should coordinate for consistency.

Reply:
It looks like Yaakov's test is consistent with example apps' start up tests. However, other tests use a different pattern. For example, the test named `Windows platform_channel_sample_test_windows` has a task named `platform_channel_sample_test_windows`. I'll match that